src/opt.cpp

```diff
@@ -482,6 +482,10 @@ void Optimize(llvm::Module *module, int optLevel) {
     }
     optPM.add(llvm::createIndVarSimplifyPass());
 
+#if ISPC_LLVM_VERSION > ISPC_LLVM_11_0
```

Comment:
This needs to be >= LLVM 12 to handle correctly LLVM 11.1

```suggestion
#if ISPC_LLVM_VERSION >= ISPC_LLVM_12_0
```

Reply:
Done